src/blocks/discord/embed/create_embed_then_add_blankfield.js

```diff
@@ -0,0 +1,33 @@
+import * as Blockly from "blockly/core";
+import { registerRestrictions } from "../../../restrictions";
+
+const blockName = "s4d_create_embed_then_add_blankfield";
+
+const blockData = {
+    "message0": "%{BKY_CREATE_EMBED_THEN_ADD_BLANKFIELD}",
+    "args0": [],
+    "colour": "#40BF4A",
+    "previousStatement": null,
+    "nextStatement": null,
+    "tooltip": "",
+    "helpUrl": ""
+};
+
+Blockly.Blocks[blockName] = {
+    init: function() {
+        this.jsonInit(blockData);
+    }
+};
+
+Blockly.JavaScript[blockName] = function() {
+    const code = `embed.addField('‎‎‎‎‎‎‎‎', '‎‎');\n`;
```

Comment:
this will work:
```suggestion
    const code = `embed.addField('\u200b‎‎‎‎‎‎‎‎', '\u200b‎‎');\n`;
```

Reply:
i tried but as it has a backslash in the front, it does not accept
and it will also not accept if you put nothing

so I did a trick : I added a blank character so it is not empty but it is also blank